sip/sip-007-stacking-consensus.md

```diff
@@ -534,21 +534,26 @@ Where `total_block_commit_amount` is equal to the sum of outputs [1, M+1].
 After the sunset phase _ends_ (i.e., blocks >= 500,000th burn block), block commits are _only_ burns, with
 a single burn output at index 1.
 
-## Stacking Operations on Bitcoin
+## STX Operations on Bitcoin
 
-As described above, PoX allows stackers to submit `stack-stx` operations on Bitcoin as well as on
-the Stacks blockchain. Any such operations must be evaluated by miners before beginning execution of
-any anchor block following those operations.
+As described above, PoX allows stackers to submit `stack-stx`
+operations on Bitcoin as well as on the Stacks blockchain. The Stacks
+chain also allows addresses to submit STX transfers on the Bitcoin
+chain. Such operations are only evaluated by the miner of an anchor block
+elected in the burn block that immediately follows the burn block that included the
+operations. For example, if a TransferStxOp occurs in burnchain block 100, then the
```

Comment:
Can you put `TransferStxOp` in code-quotes?

Reply:
Yep -- https://github.com/blockstack/stacks-blockchain/pull/2129/commits/b4c7b8878ad442a81a5fe47b6e94de287de49305